vars/tagGit.groovy

```diff
@@ -31,7 +31,11 @@ def call(Map <String, ?> config = [:]) {
       ) {
         node(POD_LABEL) {
           container('git') {
-
+            
+            if(config.addSSHKey){
```

Comment:
I think this could be made a little more generic if we had `git_server` as a variable. that way someone using this could request the `gitlab.com` keys to be added. ;)

Reply:
please note I mean `git_server` **instead** of `addSSHKey` ;)